src/cli/opts.rs

```diff
@@ -249,6 +250,23 @@ pub enum Command {
         /// The destination address receiving the coins.
         destination_address: XmrAddress,
     },
+
+    /// Output shell completion code for the specified shell (bash, zsh or fish)
```

Comment:
I think `(bash, zsh, fish, etc)` would describe the options better. Or listening all shells in `src/cli/command.rs`

`This can be done by sourcing it from the .bash_profile.` only applies for bash I think. For example, in fish, there is no need to source anything, only adding a `<binary-name>.fish` file containing the completions to  `~/.config/fish/completions/` is sufficient.

Reply:
Agree, but at the same time I wasn't sure how much detailed this should be. Do you want to propose an alternative text?